guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,44 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Allow user to run Verify Content Checksum using Hammer, on container repositories
+Added option '--validate-contents true/false' to "hammer repository synchronize" command.
+
+=== Add ACS bulk refresh option, to refresh all ACS's
+Added option '--all' to "hammer alternate-content-source refresh" command.
+ACS bulk refresh through API silently sanitizes input ids.
+
+=== Add activation key details top bar
+When the setting 'Show Experimental Labs' is enabled, the Activation Key 
+details page will be replaced by the beginnings of a redesigned page.
+
+=== Content Views and CVV enhancements to filters and refresh
+Content View filter creation/edit button now says 'Save' instead of 'Edit Rule'.
+Content view dependency solving tracked for needs_publish.
+
+=== Pulpcore and plugins upgraded to Pulpcore 3.28
+Repositories now use pulp_deb 3.0.0.
+Katello 4.9 used Puplcore 3.22, some features since 3.23.
+GCP support [3.23].
+Added a repository exact and in filters to the Distributions API [3.23].
+Added ability to filter resources by list of HREFs or IDs with pulp_href__in and pulp_id__in [3.24].
+Added commands and documentation for db encryption key rotation [3.24].
+Added filters with_content and latest_with_content to Repository list endpoint [3.25].
+Updated Django version to 4.2 LTS branch [3.25].
+Enabled OpenTelemetry metrics and tracing for the pulp-api app [3.26].
+Expose user who dispatched a Task under new field created_by [3.27].
+Added hidden field to Distributions [3.27].
+Added support to pull-through caching for streaming metadata files [3.28].
+Added support to pull-through caching for plugins with multi-artifact content types [3.28].
+Added Distribution.content_headers_for() to let plugins affect content-app response headers [3.28].
+
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-[id="katello-deprecations"]
-== Deprecations
+=== Katello-Agent removal
+Katello-Agent has been removed from Katello and is no longer available via web UI, API, or CLI.
+katello-agent will be removed by upgrading, REX alternative will not be installed as part of the upgrade.
```

Comment:
```suggestion
For patching hosts going forward, use remote execution.
katello-agent will be removed by upgrading.
Migrate to remote execution before upgrading, as REX will not be automatically enabled as part of the upgrade.
```

Reply:
I'd say you don't need to enable REX before updating. REX is always installed with Katello, but it's not on every content proxy. You can do that after you upgrade though.

You also need to make sure the hosts are updated though. I'm not sure you can do that via katello-agent today. If you can, that's a reason to do it before upgrading Katello. Otherwise I don't see a difference.

Reply:
It's true that it won't break anything if you wait to enable REX until after upgrading. But on the other hand, you won't be able to patch your hosts until you enable REX. So "Migrate before upgrading" is more of a recommendation.

Reply:
Maybe this?

```
Remote execution will not be automatically enabled as part of the upgrade. To ensure uninterrupted host patching, migrate to remote execution before upgrading.
```

Reply:
We have a guide on that as well. It'd be good to link to it.

And a minor preference to do it after upgrading is that it's trickier to link between versions, while we have infrastructure in place to link within the same version. You could argue it's within the same upgrade window, but technically it would be correct that there's less downtime in ability to patch if you perform it on the previous version.

Reply:
@ekohl @jeremylenz do you have a link to the latest guide on migrating to REX?

Reply:
It's here: https://docs.theforeman.org/nightly/Managing_Hosts/index-katello.html#Migrating_Hosts_From_Katello_Agent_to_Remote_Execution_managing-hosts

but I'm not sure of the correct way to link it via ASCIIdoc.

Also, if the docs were properly updated for Katello 4.10, that section wouldn't even be there, so I'm not quite sure what to do here.

Reply:
https://github.com/theforeman/foreman-documentation/blob/3.8/guides/common/modules/proc_migrating-hosts-from-katello-agent-to-remote-execution.adoc